specification/network/resource-manager/Microsoft.Network/stable/2024-07-01/networkSecurityPerimeter.json

```diff
@@ -1910,11 +1910,6 @@
           "description": "Resource type."
         }
       },
-      "allOf": [
-        {
-          "$ref": "./network.json#/definitions/Resource"
-        }
-      ],
       "description": "Resource that is onboarded to use network security perimeter. Also referred as perimeter associable resource."
```

Comment:
This is a good change: it's removing duplicate definitions. But why not go farther and use v6 common-types ProxyResource definition (and remove the locally defined properties (name, id, type))?